crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-typed-array/output.mjs

```diff
@@ -24,5 +24,8 @@ import "core-js/modules/es.typed-array.some.js";
 import "core-js/modules/es.typed-array.sort.js";
 import "core-js/modules/es.typed-array.subarray.js";
 import "core-js/modules/es.typed-array.to-locale-string.js";
+import "core-js/modules/es.typed-array.to-reversed.js";
+import "core-js/modules/es.typed-array.to-sorted.js";
 import "core-js/modules/es.typed-array.to-string.js";
+import "core-js/modules/es.typed-array.with.js";
```

Comment:
It's strange that it's added here since [I see just `corejs: 3`](https://github.com/nissy-dev/swc/blob/fix-7001-re/crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-typed-array/options.json#L7) without specifying a minor version where those modules were added.

Reply:
I reverted 👍